charts/keycloakx/templates/statefulset.yaml

```diff
@@ -34,35 +34,32 @@ spec:
         {{- end }}
       labels:
         {{- include "keycloak.selectorLabels" . | nindent 8 }}
-        {{- if and .Values.postgresql.enabled (and .Values.postgresql.networkPolicy .Values.postgresql.networkPolicy.enabled) }}
-        {{ include "keycloak.postgresql.fullname" . }}-client: "true"
```

Comment:
I think configuring the network policy is now out of scope of this helm chart. Users should add a custom label for this purpose if they configure such a policy on the database helm chart.

Reply:
Yes this makes sense!